Dockerfile

```diff
@@ -35,10 +35,11 @@ add CMakeLists.txt /usr/src/souper/CMakeLists.txt
 add docs /usr/src/souper/docs
 add include /usr/src/souper/include
 add lib /usr/src/souper/lib
+add runtime /usr/src/souper/runtime
+add scripts /usr/src/souper/scripts
```

Comment:
new files need to be attached to docker image, also rearranged the list with alphabet order